gitlab-pipeline/stage/trigger-images.yml

```diff
@@ -1,15 +1,11 @@
-.trigger_template:
-  stage: trigger
-  rules:
-    # Disabled for the time being. See QA-322
-    - when: never
-    # - if: '$INTEGRATION_REV =~ /^[0-9]+\.[0-9]+\.[0-9]+(-build\d+)?$/'
-    # # the following is to prevent an endless loop of qa pipelines caused by downstream pipelines
-    # - if: '$CI_PIPELINE_SOURCE == "pipeline"'
-    #   when: never
-
 trigger:mender-convert:
-  extends: .trigger_template
+  stage: trigger-images
+  needs:
+    - job: trigger:mender-dist-packages
+      artifacts: false
+  rules:
+    - if: $BUILD_MENDER_CONVERT == "true"
+    - if: $PUBLISH_RELEASE_AUTOMATIC == "true"
```

Comment:
Suggest removing this special case and relying on `BUILD_MENDER_CONVERT` instead. We can enable it explicitly when selecting "enable release" in the release tool. This way it's more explicit, and you can adjust it if you have special needs for one particular build, instead of always having it on when `PUBLISH_RELEASE_AUTOMATIC == "true"`.

Reply:
I agree, thanks.